ui/src/services/NodeUtils.js

```diff
@@ -141,7 +142,19 @@ export const getNodeListData = createSelector(
 //   control_plane: { ip: '10.0.1.42', interface: 'eth1'}
 //   workload_plane: { ip: '10.100.0.2', interface: 'eth3'},
 // }
-export const nodesCPWPIPsInterface = (IPsInterfacesObject) => {
+export const nodesCPWPIPsInterface = (
+  IPsInterfacesObject: IPInterfaces | boolean,
+): {
+  control_plane: { ip: string, interface: string },
+  workload_plane: { ip: string, interface: string },
```

Comment:
Maybe the example above confused you, what we return is camelCase property ^^
Let's update the example as well :) 

```suggestion
  controlPlane: { ip: string, interface: string },
  workloadPlane: { ip: string, interface: string },
```